t/135-worker-id.t

```diff
@@ -31,3 +31,36 @@ GET /lua
 --- no_error_log
 [error]
 --- skip_nginx: 3: <=1.9.0
+
```

Comment:
Need to run the `reindex` script on this test file to fix formatting inconsistencies like this.